ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -525,12 +556,12 @@ export OC="run_command_oc"
 mkdir -p /tmp/client
 export OC_DIR="/tmp/client"
 export PATH=${OC_DIR}:$PATH
-
+export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
```

Comment:
Only aws upgrade has this file, so this line should be moved to aws if condition in `update_cloud_credentials`

Reply:
ack, updated